CMakeLists.txt

```diff
@@ -881,7 +843,7 @@ if(BUILD_TESTS)
       --max-writes-ahead
       1000
       --repetitions
-      800
+      5000
```

Comment:
This seems to time out, you will need to bump up the test timeout from the current 90s for this to pass.

Reply:
Weird, I'd pushed a fix for this yesterday. This should run fine now.